### DISCUSSION TITLE
Refactor python code

### DISCUSSION THREAD

**Discussion by MGU2012** [Godfrey2712/freescout_ai_faq#6] [Q&A] [answered]
Hi @Godfrey2712, would it be possible for us to consider refactoring the python codes here as some stuff are redundant.

**Comment by Godfrey2712** [comment, marked as answer]
Sure we can, if you can create a couple of issues then I can look into them. You can as well consider creating a PR and assigning the issue to a branch for the PR, then I can review and merge. Thanks for the suggestions.
